### ISSUE TITLE
Window Features always ignored

### ISSUE THREAD

**Issue by MidnightJava** [rmariuzzo/react-new-window#56]
I can't find any way to set window features. I submit an object as the `features` property, and none of the mappings in that object are set as window features. That includes numeric values, like left or top, and boolean values like titlebar or scrollbars (I tried a boolean value and also string "yes/no"). I also tried setting the name property to "_blank"  or leaving it out, since the JS API says strWindowFeatures will be ignored if opening an existing window. No matter what I do, all window features are ignored.

**Comment by zning** [comment]
Same here. Great library with little tweaks

**Comment by dimitridumont** [comment]
Same here. Do you have an issue ?

**Comment by DerekSteinke** [comment]
So this issue likely because most modern browsers override window features by default (due to potential security risks), not an issue with the library.

[See here in the Mozilla docs for more details](https://developer.mozilla.org/en-US/docs/Web/API/Window/open#Window_features)

**Comment by MidnightJava** [comment]
Thanks for your reply. In that case, there is at least an issue with the library's API, correct? I was trying to do what the API said I could do, and it was silently ignored. If most modern browsers override this behavior, then perhaps it should be removed from the API. Or at least a note should be added saying that this will not work on most browsers.

**Comment by DerekSteinke** [comment]
I agree completely. At the least, a note in the docs should be included to make users aware of likely compatibility issues. I think there could still some cases where Window features are usable; like native applications running browser engines (stuff like Electron app). Even then, I'd think there are better options.

**Comment by rmariuzzo** [comment]
Will improve the docs with all your suggestion. Thank you all for the valuable discussion. Also, I apologize for the long reply.

**Comment by ASutrick** [comment]
Sorry to bring this old issue back up but window features are still ignored. I can call window.open manually and pass a string like `left=${1920},top=${-421},width=${1080},height=${1920},fullScreen=yes` and it functions as intended. 

it appears that the `center` prop and its associated logic is what's causing the issue. I have made and published a [fork](https://github.com/ASutrick/react-new-window) to remove that prop and logic (i didnt use it anyway) and now window placement works fine.

if this package is still maintained please consider reworking the center prop.

**Comment by rfrankdisw** [comment]
> Sorry to bring this old issue back up but window features are still ignored. I can call window.open manually and pass a string like `left=${1920},top=${-421},width=${1080},height=${1920},fullScreen=yes` and it functions as intended.
> 
> it appears that the `center` prop and its associated logic is what's causing the issue. I have made and published a [fork](https://github.com/ASutrick/react-new-window) to remove that prop and logic (i didnt use it anyway) and now window placement works fine.
> 
> if this package is still maintained please consider reworking the center prop.

Thank you! Confirming this fork works as described. I would love to see `center` and `features` refactored to play nice in the main repo.
